detekt/wrapper/BUILD

```diff
@@ -1,14 +1,8 @@
-load("@rules_java//java:defs.bzl", "java_binary", "java_library", "java_proto_library", "java_test")
-load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@rules_java//java:defs.bzl", "java_binary", "java_library", "java_test")
 
-proto_library(
-    name = "worker_protocol_proto",
-    srcs = ["src/main/proto/worker_protocol.proto"],
-)
-
-java_proto_library(
+alias(
     name = "worker_protocol_java_proto",
-    deps = [":worker_protocol_proto"],
+    actual = "@bazel_tools//src/main/protobuf:worker_protocol_java_proto",
```

Comment:
@Bencodes, is there some sort of documentation on this or a `BUILD` definition in the Bazel repo? I see [usages](https://github.com/bazelbuild/bazel/search?q=worker_protocol_java_proto&type=code) but no definition.

Reply:
I'm not sure about the documentation, but the definitions are here https://github.com/bazelbuild/bazel/blob/master/src/main/protobuf/BUILD#L27-L40. Little misleading because of the for loop.

Reply:
Got it, thanks. Definitely feels hidden. If you are brave and patient enough — do you want to update Bazel docs to mention these targets? Might be useful for rule developers.